src/passkit.cs

```diff
@@ -1873,4 +1958,113 @@ enum PKRadioTechnology : ulong {
 		Nfc = 1 << 0,
 		Bluetooth = 1 << 1,
 	}
+
+	[Watch (8,0), iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface PKDateComponentsRange : NSCopying, NSSecureCoding
+	{
+		[Export ("initWithStartDateComponents:endDateComponents:")]
+		[return: NullAllowed]
+		IntPtr Constructor (NSDateComponents startDateComponents, NSDateComponents endDateComponents);
+
+		[Export ("startDateComponents", ArgumentSemantic.Copy)]
+		NSDateComponents StartDateComponents { get; }
+
+		[Export ("endDateComponents", ArgumentSemantic.Copy)]
+		NSDateComponents EndDateComponents { get; }
+	}
+
+	[Watch (8,0), iOS (15,0), Mac (12,0), MacCatalyst (15,0)]
+	[BaseType (typeof(PKPaymentSummaryItem))]
+	[DisableDefaultCtor]
+	interface PKDeferredPaymentSummaryItem
+	{
+		[Export ("deferredDate", ArgumentSemantic.Copy)]
+		NSDate DeferredDate { get; set; }
+	}
+
+	[Watch (8,0), Mac (12,0), iOS (15,0)]
+	[Native]
+	public enum PKShippingContactEditingMode : ulong
+	{
+		Enabled = 1,
+		StorePickup,
+	}
+
+	[Watch (3,0), iOS (9,0)]
+	[Native]
+	public enum PKPaymentSummaryItemType : ulong
+	{
+		Final,
+		Pending,
+	}
```

Comment:
^ ideally avoid moving types - it makes reviewing harder (and my earlier comment about a breaking change likely incorrect)

Reply:
oops! sorry - as mentioned in the other comment, the diffs removed and re-added them later which threw me off-course here